src/main.cpp

```diff
@@ -126,8 +170,13 @@ void loop()
   {
     sampleAd8318();
 
-    drawPlot();
-
+    if (InputWithinBounds()) {
+      drawPlot();
+    }
+    else {
+      drawOverpowerWarning();
```

Comment:
How about still drawing the plot and showing mW/db, but display over power warning in the middle of the screen.  It is still useful to see the info for debugging purposes.

Reply:
Can do I suppose